[2024-07-16 02:46:32,151] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2021-10-03T00:00:00+00:00 [queued]>
[2024-07-16 02:46:32,457] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2021-10-03T00:00:00+00:00 [queued]>
[2024-07-16 02:46:32,459] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-07-16 02:46:32,460] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-07-16 02:46:32,471] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-07-16 02:46:32,577] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-10-03T00:00:00+00:00
[2024-07-16 02:46:32,591] {standard_task_runner.py:52} INFO - Started process 3477 to run task
[2024-07-16 02:46:32,611] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', '2021-10-03T00:00:00+00:00', '--job-id', '955', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpvtljof_y', '--error-file', '/tmp/tmpflfu7o68']
[2024-07-16 02:46:32,625] {standard_task_runner.py:77} INFO - Job 955: Subtask second_task
[2024-07-16 02:46:32,826] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.second_task 2021-10-03T00:00:00+00:00 [running]> on host f3060829e7b4
[2024-07-16 02:46:32,967] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-03T00:00:00+00:00
[2024-07-16 02:46:32,969] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-07-16 02:46:32,972] {bash.py:158} INFO - Running command: echo hello world, this is the second task, it will run after first task
[2024-07-16 02:46:32,994] {bash.py:169} INFO - Output:
[2024-07-16 02:46:32,996] {bash.py:173} INFO - hello world, this is the second task, it will run after first task
[2024-07-16 02:46:33,000] {bash.py:177} INFO - Command exited with return code 0
[2024-07-16 02:46:33,122] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20211003T000000, start_date=20240716T024632, end_date=20240716T024633
[2024-07-16 02:46:33,411] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-16 02:46:33,439] {local_task_job.py:146} INFO - Task exited with return code 0
